frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTable.component.tsx

```diff
@@ -0,0 +1,277 @@
+import { theme } from '@/v5/ui/routes/viewer/theme';
+import { combineSubscriptions } from '@/v5/services/realtime/realtime.service';
+import { enableRealtimeContainerNewTicket, enableRealtimeContainerUpdateTicket, enableRealtimeFederationNewTicket, enableRealtimeFederationUpdateTicket } from '@/v5/services/realtime/ticket.events';
+import { TicketContextComponent } from '@/v5/ui/routes/viewer/tickets/ticket.context';
+import { isCommenterRole } from '@/v5/store/store.helpers';
+import { TicketsTableContent } from './ticketsTableContent/ticketsTableContent.component';
+import { useSearchParam } from '../../../../useSearchParam';
+import { DashboardTicketsParams, TICKETS_ROUTE, VIEWER_ROUTE } from '../../../../routes.constants';
+import { ContainersAndFederationsFormSelect } from '../selectMenus/containersAndFederationsFormSelect.component';
+import { GroupByFormSelect } from '../selectMenus/groupByFormSelect.component';
+import { TemplateFormSelect } from '../selectMenus/templateFormSelect.component';
+import { Link, FiltersContainer, NewTicketButton, SelectorsContainer, SearchInput, SidePanel, SlidePanelHeader, OpenInViewerButton, FlexContainer, CompletedChip } from '../tickets.styles';
+import { GROUP_BY_URL_PARAM_TO_TEMPLATE_CASE, NONE_OPTION, hasRequiredViewerProperties } from './ticketsTable.helper';
+import { NewTicketMenu } from './newTicketMenu/newTicketMenu.component';
+import { NewTicketSlide } from '../ticketsList/slides/newTicketSlide.component';
+import { TicketSlide } from '../ticketsList/slides/ticketSlide.component';
+import { useSelectedModels } from './newTicketMenu/useSelectedModels';
+
+type FormType = {
+	containersAndFederations: string[],
+	template: string,
+	groupBy: string,
+};
+export const TicketsTable = () => {
+	const history = useHistory();
+	const params = useParams<DashboardTicketsParams>();
+	const { teamspace, project, groupBy: groupByURLParam, template: templateURLParam, containerOrFederation } = params;
+	const [modelsIds, setModelsIds] = useSearchParam('models');
+	const models = useSelectedModels();
+	const { getState } = useStore();
+	const formData = useForm<FormType>({
+		defaultValues: {
+			containersAndFederations: modelsIds?.split(',') || [],
+			template: templateURLParam,
+			groupBy: GROUP_BY_URL_PARAM_TO_TEMPLATE_CASE[groupByURLParam] || NONE_OPTION,
+		},
+	});
+	const { containersAndFederations, groupBy, template } = formData.watch();
+
+	const ticketsWithModelIdAndName = TicketsHooksSelectors.selectTicketsByContainersAndFederations(containersAndFederations);
+	const templates = ProjectsHooksSelectors.selectCurrentProjectTemplates();
+	const selectedTemplate = ProjectsHooksSelectors.selectCurrentProjectTemplateById(template);
+	const [sidePanelModelId, setSidePanelModelId] = useState<string>(null);
+	const [sidePanelTicket, setSidePanelTicket] = useState<Partial<ITicket>>(null);
+	const [showCompleted, setShowCompleted] = useState(false);
```

Comment:
I think completed should be part of the url. To change that is as easy as changing this to use  useSearchParam instead of state, right? (in this case you could modify usesearhparam to receive a conversion func just to be suere is a boolean)